src/website/views/components/toolbar/_examples.pug

```diff
@@ -449,69 +419,41 @@ h2 Examples
-                  select(class="chi-select", id="example-ba4")
-                    option(value="") Select
-                    option Option 1
-                    option Option 2
-                    option Option 3
-                button.chi-button.-icon.-flat(id="button-br-filters" aria-label="Filters" data-tooltip="Filters" data-position="top")
-                  .chi-button__content
-                    i.chi-icon.icon-filter(aria-hidden="true")
-              .chi-toolbar__filters-mobile
-                button#drawer-trigger-filter1.chi-button.-icon.-flat.chi-drawer__trigger(data-target='#drawer-1', aria-label="Button action")
-                  .chi-button__content
-                    i.chi-icon.icon-filter(aria-hidden="true")
-          .chi-toolbar__end
-            .chi-toolbar__actions
-              .chi-toolbar__actions-desktop
-                button.chi-button.-icon.-flat(id="button-br-download" aria-label="Download" data-tooltip="Download" data-position="top")
-                  .chi-button__content
-                    i.chi-icon.icon-arrow-to-bottom(aria-hidden="true")
-                button.chi-button.-icon.-flat(id="button-br-refresh" aria-label="Refresh" data-tooltip="Refresh" data-position="top")
-                  .chi-button__content
-                    i.chi-icon.icon-refresh(aria-hidden="true")
-                button.chi-button.-icon.-flat(id="button-br-column-customization" aria-label="Column Customization" data-tooltip="Column Customization" data-position="top")
-                  .chi-button__content
-                    i.chi-icon.icon-table-column-settings(aria-hidden="true")
-              .chi-toolbar__actions-mobile
-                button#drawer-trigger-actions1.chi-button.-icon.-flat.chi-drawer__trigger(data-target='#drawer-2' aria-label="Button action")
-                  .chi-button__content
-                    i.chi-icon.icon-more-vert(aria-hidden="true")
-        .chi-toolbar__ribbon
-          .chi-toolbar__start
+            chi-vue-data-table-filters.-ml--2(:filters-data='toolbar.filtersData' :custom-items='toolbar.customItemsData')
+              template(v-slot:custom-one='')
+                chi-label(for='input-3') City
+                chi-text-input#input-3
+                chi-label(for='input-4') Zip Code
+                chi-text-input#input-4
+              template(v-slot:custom-two='')
+                chi-date-picker
+          template(v-slot:end='')
+            chi-vue-tooltip(message="Download")
+              chi-button(type="icon" variant="flat" alternative-text="Download")
+                i.chi-icon.icon-arrow-to-bottom(aria-hidden="true")
+            chi-vue-tooltip(message="Refresh")
+              chi-button(type="icon" variant="flat" alternative-text="Refresh")
+                i.chi-icon.icon-refresh(aria-hidden="true")
+            chi-vue-column-customization(:columns-data='toolbar.columnsData')
+      #baseRibbon
+        chi-vue-toolbar(:ribbon="true")
```

Comment:
Following this pattern, do we have 2 stacked separate Toolbars?
If so, we need to find a way to avoid it.